### ISSUE TITLE
Sample bug report

### ISSUE THREAD

**Issue by theunraveler** [theunraveler/taskwarrior-web#26]
This is an example of a good bug report. Here, I am detailing the problem I'm running in to.

**Steps to reproduce:**
1. Go to `/tasks`
2. Click on the `New task` list
...etc...

**Output of `task _version`:**
`2.1.2`

**Output of `task show`:**

```
Config Variable                 Value
------------------------------- ------------------------------------------------
_forcecolor                     no
abbreviation.minimum            2
active.indicator                *
alias._query                    export
alias.burndown                  burndown.weekly
alias.ghistory                  ghistory.monthly
alias.history                   history.monthly
alias.rm                        delete
avoidlastcolumn                 no
bulk                            3
burndown.bias                   0.666
calendar.details                sparse
calendar.details.report         list
calendar.holidays               none
calendar.legend                 yes
calendar.offset                 no
calendar.offset.value           -1
color                           off
color.active                    black on bright green
color.alternate
color.blocked                   black on white
color.blocking                  black on bright white
color.burndown.done             on green
color.burndown.pending          on red
color.burndown.started          on yellow
color.calendar.due              white on red
color.calendar.due.today        bold white on red
color.calendar.holiday          black on bright yellow
color.calendar.overdue          black on bright red
color.calendar.today            bold white on bright blue
color.calendar.weekend          white on bright black
color.calendar.weeknumber       bold blue
color.debug                     yellow
color.due                       red
color.due.today                 red
color.error                     yellow
color.footnote                  yellow
color.header                    yellow
color.history.add               black on red
color.history.delete            black on yellow
color.history.done              black on green
color.overdue                   bold red
color.pri.H                     bold white
color.pri.L
color.pri.M                     white
color.pri.none
color.project.none
color.recurring                 magenta
color.scheduled                 black on green
color.summary.background        white on black
color.summary.bar               black on green
color.sync.added                green
color.sync.changed              yellow
color.sync.rejected             red
color.tag.none
color.tagged
color.undo.after                green
color.undo.before               red
column.padding                  1
complete.all.projects           no
complete.all.tags               no
confirmation                    yes
data.location                   /Users/Jake/Dropbox/Tasks
dateformat                      m/d/Y
dateformat.annotation           m/d/Y
dateformat.edit                 m/d/Y H:N:S
dateformat.holiday              YMD
dateformat.info                 m/d/Y H:N:S
dateformat.report               m/d/Y
debug                           no
default.command                 ready
defaultheight                   24
defaultwidth                    80
dependency.confirmation         on
dependency.indicator            D
dependency.reminder             on
detection                       off
displayweeknumber               yes
dom                             on
due                             7
exit.on.missing.db              no
expressions                     on
extensions                      off
fontunderline                   yes
gc                              on
hyphenate                       on
indent.annotation               2
indent.report                   0
journal.info                    on
journal.time                    no
journal.time.start.annotation   Started task
journal.time.stop.annotation    Stopped task
json.array                      off
list.all.projects               no
list.all.tags                   no
locale
locking                         on
merge.autopush                  ask
nag                             You have more urgent tasks.
patterns                        on
recurrence.indicator            R
recurrence.limit                1
regex                           no
report.active.columns           id,project,priority,due,start.active,entry.age,-
                                description
report.active.description       Lists active tasks
report.active.filter            status:pending start.any:
report.active.labels            ID,Project,Pri,Due,Active,Age,Description
report.active.sort              due+,priority-,project+
report.all.columns              id,status,project,priority,due,end,start.active-
                                ,entry.age,description
report.all.description          Lists all pending and completed tasks
report.all.filter               status.not:deleted
report.all.labels               ID,Status,Project,Pri,Due,Completed,Active,Age,-
                                Description
report.all.sort                 entry+
report.blocked.columns          id,depends,project,priority,due,start.active,en-
                                try.age,description
report.blocked.description      Lists all blocked tasks
report.blocked.filter           status:pending depends.any:
report.blocked.labels           ID,Deps,Project,Pri,Due,Active,Age,Description
report.blocked.sort             due+,priority-,start-,project+
report.completed.columns        end,project,priority,entry.age,description,uuid
report.completed.description    Lists completed tasks
report.completed.filter         status:completed
report.completed.labels         Complete,Project,Pri,Age,Description,UUID
report.completed.sort           end+,priority-,project+
report.list.columns             id,project,priority,due,start.active,entry.age,-
                                description
report.list.description         Lists all pending tasks
report.list.filter              status:pending '(scheduled.none: or
                                scheduled.before:now )'
report.list.labels              ID,Project,Pri,Due,Active,Age,Description
report.list.sort                due+,priority-,start-,project+
report.long.columns             id,project,priority,entry,start,due,recur,due.c-
                                ountdown,entry.age,depends,tags,description
report.long.description         Lists all pending tasks
report.long.filter              status:pending
report.long.labels              ID,Project,Pri,Added,Started,Due,Recur,Countdow-
                                n,Age,Deps,Tags,Description
report.long.sort                due+,priority-,project+
report.ls.columns               id,project,priority,description
report.ls.description           Minimal listing of all pending tasks
report.ls.filter                status:pending
report.ls.labels                ID,Project,Pri,Description
report.ls.sort                  priority-,project+
report.minimal.columns          id,project,description.truncated
report.minimal.description      Minimal listing of all pending tasks
report.minimal.filter           status:pending
report.minimal.labels           ID,Project,Description
report.minimal.sort             project+,description+
report.newest.columns           id,project,priority,due,start.active,entry.age,-
                                description
report.newest.description       Shows the newest tasks
report.newest.filter            status:pending limit:10
report.newest.labels            ID,Project,Pri,Due,Active,Age,Description
report.newest.sort              id-
report.next.columns             id,project,priority,due,start.active,entry.age,-
                                urgency,description
report.next.description         Lists the most urgent tasks
report.next.filter              status:pending limit:page '(scheduled.none: or
                                scheduled.before:now )'
report.next.labels              ID,Project,Pri,Due,A,Age,Urgency,Description
report.next.sort                urgency-,due+,priority-,start-,project+
report.oldest.columns           id,project,priority,due,start.active,entry.age,-
                                description
report.oldest.description       Shows the oldest tasks
report.oldest.filter            status:pending limit:10
report.oldest.labels            ID,Project,Pri,Due,Active,Age,Description
report.oldest.sort              id+
report.overdue.columns          id,project,priority,due,start.active,entry.age,-
                                description
report.overdue.description      Lists overdue tasks
report.overdue.filter           status:pending due.before:now
report.overdue.labels           ID,Project,Pri,Due,Active,Age,Description
report.overdue.sort             due+,priority-,start-,project+
report.ready.columns            id,project,priority,due,start.active,entry.age,-
                                urgency,description
report.ready.description        Lists the most urgent tasks
report.ready.filter             status:pending limit:page wait.none:
                                '(scheduled.none: or scheduled.before:now )'
report.ready.labels             ID,Project,Pri,Due,A,Age,Urgency,Description
report.ready.sort               urgency-,due+,priority-,start-,project+
report.recurring.columns        id,project,priority,due,recur,start.active,entr-
                                y.age,description
report.recurring.description    Lists recurring tasks
report.recurring.filter         status:pending parent.any:
report.recurring.labels         ID,Project,Pri,Due,Recur,Active,Age,Description
report.recurring.sort           due+,priority-,start-,project+
report.unblocked.columns        id,depends,project,priority,due,start.active,en-
                                try.age,description
report.unblocked.description    Lists all unblocked tasks
report.unblocked.filter         status:pending depends.none:
report.unblocked.labels         ID,Deps,Project,Pri,Due,Active,Age,Description
report.unblocked.sort           due+,priority-,start-,project+
report.waiting.columns          id,project,priority,wait,entry.age,description
report.waiting.description      Lists all waiting tasks
report.waiting.filter           status:waiting
report.waiting.labels           ID,Project,Pri,Wait,Age,Description
report.waiting.sort             wait+,priority-,project+
row.padding                     0
rule.precedence.color           due.today,active,blocking,blocked,overdue,due,s-
                                cheduled,keyword.,project.,tag.,recurring,pri.,-
                                tagged,completed,deleted
search.case.sensitive           yes
shell.prompt                    task>
tag.indicator                   +
undo.style                      side
urgency.active.coefficient      4.0
urgency.age.coefficient         2.0
urgency.age.max                 365
urgency.annotations.coefficient 1.0
urgency.blocked.coefficient     -5.0
urgency.blocking.coefficient    8.0
urgency.due.coefficient         12.0
urgency.next.coefficient        15.0
urgency.priority.coefficient    6.0
urgency.project.coefficient     1.0
urgency.scheduled.coefficient   5.0
urgency.tags.coefficient        1.0
urgency.waiting.coefficient     -3.0
verbose                         yes
weekstart                       Sunday
xterm.title                     no

Some of your .taskrc variables differ from the default values.
```
